pkgs/development/libraries/flutter-engine/package.nix

```diff
@@ -0,0 +1,255 @@
+## RFC: should we use "hostPackages" to get the packages we need for "nativeBuildInputs"?
+{ lib,
```

Comment:
avoid duplicating inputs in default.nix and package.nix
maybe just merge package.nix into default.nix

also, please use nixpkgs-fmt or alejandra

Reply:
Done